src/BaseBuilder.js

```diff
@@ -112,7 +83,7 @@ class BaseBuilder {
 	/**
 	* @description Add must_not boolean queries
 	* @param {string} operation - Type of query to perform
-	* @param {TERMS|TEXT} field - Field to apply the query to
+	* @param {TERMS|TEXT|Object} field - Field to apply the query to
```

Comment:
Ok, this is fixed, you can see the enum and typdef declarations in the constants file.